.travis.yml

```diff
@@ -8,6 +8,6 @@ env:
 language: node_js
 
 node_js:
-  - "5"
+  - "6"
```

Comment:
"stable" has no meaning in semver (and thus in node/io.js >= 1.0), where everything is stable.